Mage.Sets/src/mage/sets/CrimsonVowCommander.java

```diff
@@ -136,6 +136,8 @@ private CrimsonVowCommander() {
         cards.add(new SetCardInfo("Spectral Sailor", 114, Rarity.UNCOMMON, mage.cards.s.SpectralSailor.class));
         cards.add(new SetCardInfo("Spectral Shepherd", 98, Rarity.UNCOMMON, mage.cards.s.SpectralShepherd.class));
         cards.add(new SetCardInfo("Stensia Masquerade", 150, Rarity.UNCOMMON, mage.cards.s.StensiaMasquerade.class));
+        cards.add(new SetCardInfo("Strefan, Maurer Progenitor", 2, Rarity.MYTHIC, mage.cards.s.StrefanMaurerProgenitor.class, NON_FULL_USE_VARIOUS));
+        cards.add(new SetCardInfo("Strefan, Maurer Progenitor", 40, Rarity.MYTHIC, mage.cards.s.StrefanMaurerProgenitor.class, NON_FULL_USE_VARIOUS));
```

Comment:
Again, I'm not sure what this NON_FULL_USE_VARIOUS is. Card 40 is the full art version.  I think it's best to remove this tag and remove card 40 for now.  Alt art stuff usually gets taken care of once the set is complete and image downloader code gets added for the set.

Reply:
Okay. If those get handled when the set is finished, i'll remove it from this (and the other cards I'm adding)

Reply:
Or, I guess I'll just leave it if it's already implemented. One less thing to take care off when the set is finished.

Reply:
Well, it looks like this probably correct anyway (and it'll get changed when the other alt-art cards get added if not) so it's fine.  I probably just wouldn't worry about doing this in the future.